src/chrome/content/rules/Tumblr.xml

```diff
@@ -122,7 +106,7 @@
 	are still vulnerable whenever they browse blogs while logged in. :(
 
 -->
-<ruleset name="Tumblr.com (partial)">
```

Comment:
This rule is still `partial` because every user has it's own subdomain which is redirected to http (please document this behavior on the top).